README.md

```diff
@@ -107,7 +107,8 @@ docker-compose up -d
 yarn start # Run this in another terminal to keep it open
 yarn cypress:run
 
-# Manually check that content-security-policy in ./internals/serve-with-csp.json doesn't break any functionality
+# Manually check that content-security-policy in ./internals/getCsp.js doesn't break any functionality
+yarn --silent print-csp
```

Comment:
@tjanez new command. Alternatively you can copy https://github.com/oasisprotocol/oasis-wallet-web/blob/3291206233849475f45aaed1d26d319a1d8f0895/internals/getCsp.js#L6-L33